src/odemis/driver/blinkstick.py

```diff
@@ -65,15 +65,11 @@ def __init__(self, name, role, sn=None, max_power=0.1, inversed=False, **kwargs)
         time.sleep(0.1)  # Device apparently needs some time to recover
 
         self._shape = ()
-        # TODO: allow to change the power also via emissions? Or just make it a choice
-        self.emissions = model.ListVA([0], unit="", setter=self._setEmissions)
-        self.emissions.subscribe(self._updatePower)
         # list of 5-tuples of floats
         self.spectra = model.ListVA([(380e-9, 390e-9, 560e-9, 730e-9, 740e-9)],
                                     unit="m", readonly=True)
 
-        self.power = model.FloatContinuous(0., (0., self._max_power), unit="W",
-                                           setter=self._setPower)
+        self.power = model.ListContinuous([0.,], ((0.,), (max_power,)), unit="W", setter=self._setPower)
```

Comment:
For *all* these ListContinuous, you need to indicate that the cls can be an int too. Otherwise, it's really annoying in practice, to have to put a magical `.` after 0 or 1. I think you can just write: `cls=numbers.Real`. Otherwise, you'll have to do like what we've done for a lot of TupleContinous: `(int, long, float)`, and import `long` from builtins.past, to be compatible with Python 2 and 3.